composer.json

```diff
@@ -13,11 +13,14 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.1",
         "symfony/config": "^2.3 || ^3.0 || ^4.0"
```

Comment:
Time to go up to `^2.7 || ^3.4 || ^4.0` here?

Reply:
Yeah, fair.